Sources/Mistica/Components/Lists/ListCellContentView.swift

```diff
@@ -46,7 +46,7 @@ open class ListCellContentView: UIView {
     @frozen
     public enum CellAssetType: Equatable {
         case none
-        case image(UIImage)
+        case image(UIImage, size: CGSize? = nil)
```

Comment:
not sure if we should allow any size or just force some aspect ratios like:
```suggestion
        case image(UIImage, ImageAspectRatio?)
```
```swift
enum ImageAspectRatio {
        case height(CGFloat, ImageAspectRatioType)
        case width(CGFloat, ImageAspectRatioType)
}

enum ImageAspectRatioType {
        case oneOne
        case sixteenNine
        case sevenTen
}
```

Maybe it should be decided by Design team

Reply:
Maybe we could start with static types and if in the future it is necessary we can add a `custom`case. 
```
enum ImageAspectRatioType {
     case oneOne
     case sixteenNine
     case sevenTen
     case custom(width: CGFloat, height: CGFloat)
}
```